test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/JsonNoMetadataLevelTests.cs

```diff
@@ -13,14 +13,21 @@ namespace Microsoft.OData.Tests.JsonLight
 {
     public class JsonNoMetadataLevelTests
     {
-        private readonly JsonNoMetadataLevel testSubject = new JsonNoMetadataLevel();
+        private readonly JsonNoMetadataLevel testSubject = new JsonNoMetadataLevel(/* alwaysAddTypeAnnotationsForDerivedTypes */ false);

```

Comment:
>JsonNoMetadataLevel [](start = 63, length = 19)

Maybe add an "Ctor" overload without alwaysAddTypeAnnotationsForDerivedTypes parameter and call "ctor" with alwaysAddTypeAnnotationsForDerivedTypes using "false".

Reply:
Additional constructor added in the 3rd commit to this PR.